docs/Advanced-Concepts/Chip-Communication.rst

```diff
@@ -130,13 +118,81 @@ Using the JTAG Interface
 ------------------------
 
 The main way to use JTAG with a Rocket Chip based system is to instantiate the Debug Transfer Module (DTM)
-and configure it to use a JTAG interface. The default Chipyard designs instantiate the DTM and configure it
-to use JTAG. You may attach OpenOCD and GDB to any of the default JTAG-enabled designs.
+and configure it to use a JTAG interface.
+The default Chipyard designs instantiate the DTM and configure it to use JTAG.
+You may attach OpenOCD and GDB to any of the default JTAG-enabled designs.
 
 Debugging with JTAG
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~
 
-Please refer to the following resources on how to debug with JTAG.
+Please refer to the following resources on how to debug a Rocket Chip system with JTAG.
 
 * https://github.com/chipsalliance/rocket-chip#-debugging-with-gdb
 * https://github.com/riscv/riscv-isa-sim#debugging-with-gdb
```

Comment:
Should we move these lines to be after step 2, so people don't jump directly to rocketchip and try to go by the full rocketchip instructions, but rather start by building the chipyard configs and running the chipyard simulation, and then using the rocketchip instructions only for OpenOCD and GDB instructions?

Reply:
I reworked this a bit. Check now.